webapp/package.json

```diff
@@ -29,10 +29,12 @@
     "jquery": "^3.6.0",
     "js-base64": "^3.6.1",
     "less-vars-to-js": "^1.3.0",
+    "node-sass": "^4.0.0",
```

Comment:
@enicolasgomez is this a consequence of branching from development?
This should've been removed since not used anyamore